task/sast-snyk-check/0.2/MIGRATION.md

```diff
@@ -1,4 +1,12 @@
 # Migration from 0.1 to 0.2
 
+Version 0.2:
+
+These changes originates from ProdSec requirements in order for sast snyk check to scan dependencies. See https://issues.redhat.com/browse/STONEINTG-1020 for more information.
+Inherited from sast-snyk-check task,
```

Comment:
```suggestion
```

Reply:
Thanks!
Indeed there is no action required from users, but we were approached by users because of warning icon in renovate PR https://github.com/stolostron/must-gather/pull/260
So I wanted to reflect that here.

Reply:
Do you think that the action from user should be like this?

**Renovate bot PR will be created with warning icon for a sast-snyk-check which is expected, no action from users are required.**

Reply:
That's much better, thanks!